android/modules/calendar/src/java/ti/modules/titanium/calendar/EventProxy.java

```diff
@@ -243,6 +244,36 @@ public static EventProxy createEvent(CalendarProxy calendar, KrollDict data)
 		}
 	}
 
+	private AttendeeProxy[] getAttendeeProxies() {
+		final String[] attendeeProjection = new String[]{
+				CalendarContract.Attendees._ID,
+				CalendarContract.Attendees.EVENT_ID,
+				CalendarContract.Attendees.ATTENDEE_NAME,
+				CalendarContract.Attendees.ATTENDEE_EMAIL,
+				CalendarContract.Attendees.ATTENDEE_TYPE,
+				CalendarContract.Attendees.ATTENDEE_RELATIONSHIP,
+				CalendarContract.Attendees.ATTENDEE_STATUS
+		};
+		final String query = "(" + CalendarContract.Attendees.EVENT_ID + " = ?)";
+		final String[] args = new String[]{id};
+		ContentResolver contentResolver = TiApplication.getInstance().getContentResolver();
+		final Cursor cursor = contentResolver.query(CalendarContract.Attendees.CONTENT_URI, attendeeProjection, query, args, null);
+		int index = 0;
+		AttendeeProxy[] result = new AttendeeProxy[cursor.getCount()];
+		while (cursor.moveToNext()) {
+			//create a proxy instance
+			AttendeeProxy proxyForRow = new AttendeeProxy(cursor.getString(cursor.getColumnIndex(CalendarContract.Attendees.ATTENDEE_EMAIL)),
+															cursor.getString(cursor.getColumnIndex(CalendarContract.Attendees.ATTENDEE_NAME)),
+															cursor.getInt(cursor.getColumnIndex(CalendarContract.Attendees.ATTENDEE_TYPE)),
+															cursor.getInt(cursor.getColumnIndex(CalendarContract.Attendees.ATTENDEE_STATUS)),
+															cursor.getInt(cursor.getColumnIndex(CalendarContract.Attendees.ATTENDEE_RELATIONSHIP))
```

Comment:
I looked up our docs. We do define UNKNOWN types in iOS. We need to do the same on Android for types we don't recognize as well. Perhaps that should be determined in the `AttendeeProxy`? I'll leave that up to you.
http://docs.appcelerator.com/platform/latest/#!/api/Titanium.Calendar